client/server/render/index.js

```diff
@@ -98,7 +104,7 @@ function render( element, key, req ) {
 				config.isEnabled( 'ssr/always-log-cache-misses' )
 			) {
 				// Log 0.1% of cache misses
-				logToLogstash( {
+				req.logger.warn( {
```

Comment:
I took the opportunity to move to the normal logger here.

Reply:
Does it log to logstash, too?